### PR TITLE
haskell-update: Mark essence-of-live-coding* as unbroken

### PR DESCRIPTION
**Pull request by turion**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/haskell-modules/configuration-hackage2nix.yaml [hunk 1]
```diff
@@ -734,10 +734,6 @@ default-package-overrides:
   - errors-ext ==0.4.2
   - ersatz ==0.4.8
   - esqueleto ==3.3.3.2
-  - essence-of-live-coding ==0.1.0.3
-  - essence-of-live-coding-gloss ==0.1.0.3
-  - essence-of-live-coding-pulse ==0.1.0.3
-  - essence-of-live-coding-quickcheck ==0.1.0.3
```
Comment (cdepillabout) [inline pkgs/development/haskell-modules/configuration-hackage2nix.yaml:740]: @turion Did you mean to delete these?

These set the package versions for `essence-of-live-coding-*` to the versions from the Stackage LTS (and shouldn't be deleted).

Comment (turion) [reply pkgs/development/haskell-modules/configuration-hackage2nix.yaml:740]: Oh, I misunderstood that then. Thanks for pointing out! I always have trouble finding my way around in this very long file.

I guess one could refactor it and put the list of broken packages in a separate file, so one always finds them.

Comment (turion) [reply pkgs/development/haskell-modules/configuration-hackage2nix.yaml]: I reverted this.

**Review by cdepillabout — APPROVED** [review]
Everything compiles, thanks!

**Pull request merged**